docs/01-usage/plugins.md

```diff
@@ -80,11 +75,11 @@ Global plugin options can defined at the root of the **semantic-release** config
     "@semantic-release/commit-analyzer",
     "@semantic-release/release-notes-generator",
     ["@semantic-release/github", {
-      "assets": ["dist/**"]
+      "assets": ["dist/**"]                # plugin configuration
```

Comment:
Unfortunately, comments are not allowed in JSON. Adding them will make it harder to copy & paste the examples.

We could add a description below explaining the different bits of it?

Reply:
Done in next push. Explanation already in the text.